### PR TITLE
AccountForm: improve spacing and remove button borders

### PR DESCRIPTION
**Pull request by GVodyanov**
Fix  https://github.com/nextcloud/mail/issues/9915

| After | Before |
| - | - |
| Screenshot from 2024-07-26 10-04-32 | image |

### CODE REVIEW

File: src/components/AccountForm.vue [hunk 1]
```diff
@@ -703,6 +703,19 @@ export default {
 	font-weight: bold;
 }
 
+:deep(.checkbox-radio-switch) {
+	border: 0 !important;
```
Comment (ChristophWurst) [inline src/components/AccountForm.vue:707]: this border is indended: https://nextcloud-vue-components.netlify.app/#/Components/NcCheckboxRadioSwitch

we have to fix that it's intermitted

File: src/components/AccountForm.vue [hunk 2]
```diff
@@ -703,6 +703,19 @@ export default {
...
+}
+
+:deep(.input-field) {
+	margin-bottom: calc(var(--default-grid-baseline) * 3);
+	margin-top: var(--default-grid-baseline);
+}
+
+label {
+	margin-top: calc(var(--default-grid-baseline) * 2);
```
Comment (ChristophWurst) [inline src/components/AccountForm.vue:716]: as an alternative we could convert every HTML label+input into NcInput. It is even more compact

File: src/components/AccountForm.vue (hunk 1 above)
Comment (GVodyanov) [reply src/components/AccountForm.vue:707]: Yep

File: src/components/AccountForm.vue (hunk 2 above)
Comment (GVodyanov) [reply src/components/AccountForm.vue:716]: Done

**Comment by GVodyanov** [comment]
Here's how it's looking now 
image

**Pull request merged**